build/common.props

```diff
@@ -20,4 +20,7 @@
     <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(RoslynVersion)" PrivateAssets="All" />
```

Comment:
We'd like to use the new compiler for single repo builds but not for coherence-signed builds.

/cc @pranavkm @natemcmaster